cheat_sheets/pyprimemesh_cheat_sheet/pyprimemesh_cheat_sheet.tex

```diff
@@ -12,7 +12,7 @@
 \raggedright
 \footnotesize
 
-PyPrimeMesh Cheat Sheet
+% PyPrimeMesh Cheat Sheet
 % ----------------------------------------
```

Comment:
Are you sure about this?

Reply:
Yes, it is rendering on the top left, if you see here , https://[cheatsheets.docs.pyansys.com/pyprimemesh_cheat_sheet.pdf](https://cheatsheets.docs.pyansys.com/pyprimemesh_cheat_sheet.pdf)